### DISCUSSION TITLE
Header Logo

### DISCUSSION THREAD

**Discussion by Elwafi-Elmehdi** [chrismwilliams/astro-theme-cactus#48] [Q&A] [answered]
Hey @chrismwilliams, I hope you are fine, I just want your advice on the SVG header logo, been one month since I started experimenting with logos and I friend of mine designed one for me a logo but I could not get it to fit correctly in the content box.

You can inspect the code if you want [My repo](https://github.com/Elwafi-Elmehdi/mehdi4j) 

### SVG Logo
![icon](https://user-images.githubusercontent.com/77673506/217941048-c24bc874-3673-44b8-84f1-84bbaa8b833f.svg)

### Logo of website preview, favicon
![mehdi4j](https://user-images.githubusercontent.com/77673506/217941064-bf956c6f-ca70-4389-8b08-9221be016679.png)

**Comment by chrismwilliams** [comment]
Hi @Elwafi-Elmehdi I'll try and help but I'm not quite sure what you're trying to do. Do you want to use the abov svg logo in the header [here](https://github.com/Elwafi-Elmehdi/mehdi4j/blob/fc177b5720313d4f05a82a8e7d55d516cfebf2d7/src/components/layout/Header.astro#L47) instead of the png logo?

Also, as an aside, any images you use that are imported from /public [won't be processed](https://docs.astro.build/en/guides/integrations-guide/image/#images-in-public). Basically @astro/image won't try to reduce the file size/format, not sure if that was intended.

**Comment by Elwafi-Elmehdi** [comment]
Hey @chrismwilliams, yes I am trying to change the Header, with the SVG logo but I have trouble with the aspect ratio and I could not get it right, you can check  [my blog](mehdi4j.online) the logo I am using is not nice looking.

I am sure you are right and it was not intended by me, because I know to work with neither astro nor its integrations so if you can give a simple example I really appreciate it.

**Comment by chrismwilliams** [comment, marked as answer]
Hey @Elwafi-Elmehdi hopefully [#1](https://github.com/Elwafi-Elmehdi/mehdi4j/pull/1) should help.

**Reply by Elwafi-Elmehdi** [reply]
Thank you so mush @chrismwilliams i really appreciate it.
